llvm/test/tools/repo2obj/extern_mixed_weak_global_references.ll

```diff
@@ -0,0 +1,56 @@
+// Check the weak symbol binding: the output symbol is global if one or more of the references to it are strong.
```

Comment:
This file has a .ll extension which is rather misleading (it’s JSON not LLVM-IR).

Reply:
Changed to .json.